### DISCUSSION TITLE
Featured: Custom Border Radius Configuration

### DISCUSSION THREAD

**Discussion by cyphertechschool** [saadeghi/daisyui#2936] [Ideas / feature requests]
Here , Need to add customise border-radius for all themes

**Comment by saadeghi** [comment]
Here are the CSS variables for border radius:  
https://daisyui.com/docs/utilities/#-3

And here's how to customize an existing theme:  
https://daisyui.com/docs/themes/#-7


Let me know if you have any questions

**Reply by cyphertechschool** [reply]
thankyou
